.github/workflows/build-world.yaml

```diff
@@ -0,0 +1,96 @@
+name: Build the world
+
+on:
+  workflow_dispatch:
+    
+jobs:
+  build:
+    name: Build OS packages
+    runs-on:
+      - self-hosted
+      - ${{ matrix.arch }}
+
+    # Only run one build per arch at a time
+    concurrency:
+      group: build-world-${{ matrix.arch }}
+
+    strategy:
+      matrix:
+        arch: [ "x86_64", "arm64" ]
+      fail-fast: false
+
+    steps:
+      - name: Generate snapshot date
+        id: snapshot-date
+        run: |
+          echo "date=$(date -u +%Y%m%d)" >> $GITHUB_OUTPUT
+          echo "epoch=$(date -u +%s)" >> $GITHUB_OUTPUT
+        shell: bash
+
+      - uses: actions/checkout@v3
+
+      - run: echo "${{ secrets.MELANGE_RSA }}" > ./wolfi-signing.rsa
+
+      - run: |
+          sudo mkdir -p /etc/apk/keys
+          sudo cp ./wolfi-signing.rsa.pub /etc/apk/keys/wolfi-signing.rsa.pub
+
+      - uses: chainguard-dev/actions/setup-melange@main
```

Comment:
Not to be disruptive to this change, but I wonder if we should migrate to copying this out of the SDK image as well 🤔

Reply:
afaiu `setup-melange` does a bunch of other prereq related things (ie: bwrap setup) that make it harder than just copying the binary out like we did for `wolfictl`, but if there's precedence let me know

Reply:
If this is a bare build, which isn't uploading anything, why not just use the SDK image directly?

Reply:
We do this in our regular builds too: https://github.com/wolfi-dev/os/blob/main/.github/workflows/push-production.yaml#L38

Reply:
Yes, I know.  What I am saying is that we should stop doing that, and use the SDK image instead.

Reply:
(In other words, we should avoid adding additional variations of this particular technical debt.)